test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -337,7 +337,7 @@ export default class BaseFormObject {
 
   getNthErrorMessage(fieldName, value) {
     if (userVariables.gen3) {
-      return this.el.find(`#${fieldName}-error-${value}`).innerText;
+      return this.el.find(`#${fieldName}-error`).child('div').child('ul').child('li').nth(value).innerText;
```

Comment:
Odyssey field level error component is a single parent element with each error as a child, while old SIW behavior was each error is its own element